### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Kwamay**
-[ ] Added html and css files to project
-[ ] Added README.me and License files to project
-[ ] Added Linters to project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,22 @@
...
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Prerequisites](#prerequisites)
+  - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
```
Comment (TedLivist) [inline README.md:22]: - [ ] This "Run tests" and the one below it "Deployment" are broken links. Kindly remove them or add corresponding sections for them.

File: README.md [hunk 2]
```diff
@@ -61,180 +29,104 @@ For more information on the importance of a professional README for your reposit
...
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
 > Mention all of the collaborators of this project.
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly remove this section description as it is a dummy text from the template meant for your guidance and should not be part of the final README.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Kwamay,

Good job so far!
✔️ The `test.md` file is well ignored
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Kwamay, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added run test instructions. ✔️ 
- Removed dummy text. ✔️ 
 
## [Optional] suggestions:
- For future pull requests, please notice that your branch name `hello_br` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-hello_br`. This makes it easier to track the changes and the purpose of each branch.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**